src/store/db.ts

```diff
@@ -47,7 +48,8 @@ export const getEpoch = async (arIOReadSDK: AoARIORead, epochIndex: number) => {
 
   if (epochData) {
     try {
-      await db.epochs.add(epochData);
+      console.log('Saving epoch', epochIndex, epochData)
+      await networkPortalDB.epochs.add(epochData);
     } catch (e) {
       console.error('Error with epoch data saving:', epochIndex, e);
```

Comment:
This seems like the kind of thing we want a pulse on for Sentry. Suggest pulling in `captureException` to pull this into our observation tools.

Reply:
There's an open story to audit codebase for Sentry (PE-6499). Will get introduced when that story is completed.